### PR TITLE
Audit API Static Security Analysis Enhancement

### PR DESCRIPTION
**Pull request by nameisaravind**
This PR is created to enhance the audit api static security quality check and the logic behind adding the audit statuses.

### CODE REVIEW

File: api-audit/src/main/java/com/capitalone/dashboard/evaluator/StaticSecurityAnalysisEvaluator.java [hunk 1]
```diff
@@ -64,27 +71,31 @@ private SecurityReviewAuditResponse getStaticSecurityScanResponse(CollectorItem
...
-                }
-            } else if (metric.getName().equalsIgnoreCase("Critical")) {
-                securityReviewAuditResponse.addAuditStatus(CodeQualityAuditStatus.STATIC_SECURITY_SCAN_FOUND_CRITICAL);
-            } else if (metric.getName().equalsIgnoreCase("High")) {
-                securityReviewAuditResponse.addAuditStatus(CodeQualityAuditStatus.STATIC_SECURITY_SCAN_FOUND_HIGH);
+        Set<CodeQualityMetric> metrics = returnQuality.getMetrics();
+        boolean isSecurityCritical = metrics.stream().anyMatch(metric -> metric.getName().equalsIgnoreCase(STR_CRITICAL));
+        boolean isSecurityHigh = metrics.stream().anyMatch(metric -> metric.getName().equalsIgnoreCase(STR_HIGH));
+
+        if (isSecurityCritical){
```
Comment (rvema) [inline api-audit/src/main/java/com/capitalone/dashboard/evaluator/StaticSecurityAnalysisEvaluator.java]: can we do this without multiple if else conditions? Too confusing to read

**Comment by nameisaravind** [comment]
Closing this PR for now since there is a design confirmation needed.

File: api-audit/src/main/java/com/capitalone/dashboard/evaluator/StaticSecurityAnalysisEvaluator.java (hunk 1 above)
Comment (nameisaravind) [reply api-audit/src/main/java/com/capitalone/dashboard/evaluator/StaticSecurityAnalysisEvaluator.java]: these if/else necessary to give priority over crtitical case over others.

**Comment by nameisaravind** [comment]
> Add Junit for staticSecurityAnalysisEvaluator

JUnit added for StaticSecurityAnalysisEvaluator

**Pull request merged**